mungegithub/mungers/approval-handler.go

```diff
@@ -391,6 +391,12 @@ func (h ApprovalHandler) getApprovedOwners(files []*githubapi.CommitFile, approv
 	// TODO: go through the files starting at the top of the tree
 	needsApproval := sets.NewString()
 	for _, file := range files {
+
```

Comment:
Non-nit, actually very improtant: you have two extra empty lines, one here and one at the bottom. please fix ASAP!